manual/manual.tex

```diff
@@ -34,7 +34,7 @@
 \chead{}
 \rhead[\fancyplain{}{\sl\leftmark}]{\fancyplain{}{\sl\thepage}}
 \lfoot[\fancyplain{\sl\thepage}{}]{}
-\cfoot{\ifDraft\textsf{Data61 Confidential}\fi}
+\cfoot{\ifDraft\textsf{seL4 Foundation Confidential}\fi}
```

Comment:
We can probably drop the confidential part now, since any updates will be developed publicly anyway. I.e. I'd propose:
```suggestion
\cfoot{}
```

Reply:
is says `\ifDraft` there and having drafts may be handy for all non-release builds. So use "seL4 Foundation DARFT" instead?

Reply:
IIRC, when the draft option is on, each page will have a huge DRAFT watermark in the background, so I think it's Ok to leave the footer empty.